specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/Diagnostics.json

```diff
@@ -432,6 +479,212 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{environmentName}/detectorProperties/daprApi/": {
+      "get": {
+        "tags": [
+          "ManagedEnvironments",
+          "Diagnostics"
+        ],
+        "summary": "Get the Dapr components of a Managed Environment.",
+        "description": "Get the Dapr components of a Managed Environment used to host container apps.",
+        "operationId": "ManagedEnvironmentsDiagnostics_ListDaprComponents",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DaprComponentsCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Dapr Components": {
+            "$ref": "./examples/DaprComponents_List.json"
```

Comment:
>./examples/DaprComponents_List.json [](http://example.com/codeflow?start=21&length=35)

Is the example added? It's not in this PR, same for other ones in this change

Reply:
Since it's a proxy API of another one, the response is exactly the same. We don't need an example for it

Reply:
what do you mean by proxy api ? Why are you creating duplicate APIs ?

Reply:
please address the comment about adding examples for us to sign off. Also once you have added the examples, pl remove the "ARMChangesRequested" label on the PR to make it actionable on the API reviewers